api_app/models/schemas/airlock_request.py

```diff
@@ -29,15 +29,33 @@ class Config:
         }
 
 
-class AirlockRequestInList(BaseModel):
-    airlockRequests: List[AirlockRequest] = Field([], title="Airlock Requests")
+class AirlockRequestWithAllowedUserActions(BaseModel):
+    airlockRequest: AirlockRequest = Field([], title="Airlock Request")
```

Comment:
```suggestion
    airlockRequests: AirlockRequest = Field([], title="Airlock Requests")
```

Reply:
this one can only contain 1 request

Reply:
Why do we return one in a List? I don't understand. It is not done like that anywhere else in the API

Reply:
Oh I thought it's a list. ok